docs/design/custom-tab-placement.md

```diff
@@ -0,0 +1,54 @@
+  </CustomTab>
+</ExtensionPoint>
+```
+
+Keep the following points in mind:
+
+- The  `<InsertBefore>` and  `<InsertAfter>` elements are optional. If you use neither, then your custom tab will appear as the rightmost tab on the ribbon.
+- The  `<InsertBefore>` and  `<InsertAfter>` elements are mutually exclusive. You cannot use both.
+- If the user installs more than one add-in whose custom tab is configured for the same place, say after the **Review** tab, then the tab for the most recently installed add-in will be located in that place. The tabs of the previously installed add-ins will be moved over one place. For example, the user installs add-ins A, B, and C in that order and all are configured to insert a tab after the **Review** tab, then the tabs will appear in this order: **Review**, **AddinCTab**, **AddinBTab**, **AddinATab**.
+- Users can customize the ribbon in the Office application. For example, a user can move or hide your add-in's tab. You cannot prevent this or detect that it has happened.
+- If a user moves one of the built-in tabs, then Office interprets the `<InsertBefore>` and  `<InsertAfter>` elements in terms of *the default location of the built-in tab*. For example, if the user moves the **Review** tab to the right end of the ribbon, Office will interpret the markup in the example above as meaning "put the custom tab just to the right of *where the Review tab would be by default*."
+
+## Specifying which tab has focus when the document opens
+
+Office always gives default focus to the tab that is immediately to the right of the **File** tab. By default this is the **Home** tab. If you configure your custom tab to be to before the **Home** tab, with `<InsertBefore>TabHome</InsertBefore>`, then your custom tab will have focus when the document opens.
+
+> [!IMPORTANT]
+> Giving excessive prominence to your add-in inconveniences and annoys users and administrators. Do not position a custom tab before the **Home** tab unless your add-in is the primary tool with which users will interact with the document.
+
+## Behavior on unsupported platforms
+
+If your add-in is installed on a platform that does not support [requirement set AddinCommands 1.3](../reference/requirement-sets/add-in-commands-requirement-sets.md), then the markup described in this article is ignored and your custom tab will appear as the rightmost tab on the ribbon. To prevent your add-in from being installed on platforms that don't support the markup, add a reference to the requirement set to the `<Requirements>` section of the manifest. For instructions, see [Set the Requirements element in the manifest](../develop/specify-office-hosts-and-api-requirements.md#set-the-requirements-element-in-the-manifest). Note that the runtime checks that are described in the article are not applicable to **AddinCommands 1.3** because it contains only manifest markup that is read when the add-in starts up and before a runtime check can occur.
```

Comment:
```suggestion
If your add-in is installed on a platform that doesn't support [requirement set AddinCommands 1.3](../reference/requirement-sets/add-in-commands-requirement-sets.md), then the markup described in this article is ignored and your custom tab will appear as the rightmost tab on the ribbon. To prevent your add-in from being installed on platforms that don't support the markup, add a reference to the requirement set in the `<Requirements>` section of the manifest. For instructions, see [Set the Requirements element in the manifest](../develop/specify-office-hosts-and-api-requirements.md#set-the-requirements-element-in-the-manifest). Note that the runtime checks described in the article are not applicable to **AddinCommands 1.3** because it contains only manifest markup that is read when the add-in starts up and before a runtime check can occur.
```